topics/google/index.md

```diff
@@ -0,0 +1,12 @@
+---
+display_name: Google
```

Comment:
You could add a `related: android, chrome-extension, chrome, material-design, go, google-maps` line.